Srom even nicer and even prettier

DreamySkrell opened the pull request:
![image](https://user-images.githubusercontent.com/107256943/232531242-ab91af2d-31ee-4e2e-9991-f11ccfd413a5.png)
![image](https://user-images.githubusercontent.com/107256943/232531397-4e73a17d-c214-43e5-b54f-2bedce85cbfd.png)
![image](https://user-images.githubusercontent.com/107256943/232531479-1a405c9f-7d96-4aeb-892b-9817206ffa6f.png)
![image](https://user-images.githubusercontent.com/107256943/232531579-6fd53eaa-32e8-4004-8eca-aa4b34c9f38a.png)
![image](https://user-images.githubusercontent.com/107256943/232531700-c91447ca-878d-442b-82c6-b3130d4cdbfa.png)

Mostly just lights with less power/range.

DreamySkrell commented:
!review

atteria reviewed (changes requested):
The exoplanet snow here was added before I added automatic smoothing to it, so it overlaps with the edge decals that preceded that. You should probably either delete the edge decals or swap the exoplanet snow with the holodeck snow.

DreamySkrell commented:
> The exoplanet snow here was added before I added automatic smoothing to it, so it overlaps with the edge decals that preceded that. You should probably either delete the edge decals or swap the exoplanet snow with the holodeck snow.

Deleted the edge decals.
